src/routes/staking/staking-node.tsx

```diff
@@ -71,6 +72,10 @@ export const StakingNode = ({ vegaKey, data }: StakingNodeProps) => {
     return BigNumber.sum.apply(null, [new BigNumber(0), ...amountsNextEpoch]);
   }, [currentEpoch, data?.party?.delegations, node, stakeThisEpoch]);
 
+  const pendingStakeNextEpoch = React.useMemo(() => {
+    return stakeNextEpoch.minus(stakeThisEpoch);
```

Comment:
```suggestion
    return stakeNextEpoch.minus(stakeThisEpoch).isZero();
```

Reply:
I don't understand this suggestion. `pendingStakeNextEpoch` is the amount (BigNumber) that is pending, your suggested change returns a boolean.

Reply:
Ah I see pendingStakeNextEpoch is also used on line 135. My mistake :+1: